### ISSUE TITLE
Template constructor with String doesn't work on Windows OS with classpath: type 

### ISSUE THREAD

**Issue by lauvigne** [CodeStory/code-story-http#1]
For information, 
net.codestory.http.templating.Template constructor with String doesn't work on Windows OS with classpath: type :

```
Caused by: java.nio.file.InvalidPathException: Illegal char <:> at index 9: classpath:web\partial
    at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
    at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
    at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
    at sun.nio.fs.WindowsPath.parse(WindowsPath.java:94)
    at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:255)
    at java.nio.file.Paths.get(Paths.java:84)
    at net.codestory.http.templating.Template.<init>(Template.java:32)
```

See in sun.nio.fs.WindowsPathParser.normalize :

``` java
// Reserved characters for window path name
private static final String reservedChars = "<>:\"|?*";
```

**Comment by lauvigne** [comment]
Fixed on my fork with b7e442f94403915c25eb09c6d6ec9bafe96cb232 and b07595571f5e2bd0b9c6d44071b9a6a86c67bbc8

``` java
ClassLoader.getSystemResource("web").getFile()
```

return /C:/ and sun.nio.fs.WindowsPathParser.parse want C:/

**Comment by dgageot** [comment]
Latest code doesn't use classpath: anymore
